.github/workflows/pr-labels.yml

```diff
@@ -0,0 +1,56 @@
+name: PR Label Management
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  manage-labels:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
do we need this checkout line?

Reply:
I double checked and no, we dont need that, I will correct